yarn.lock

```diff
@@ -2370,6 +2370,7 @@ __metadata:
     "@cardano-sdk/ogmios": ^0.10.0
     "@cardano-sdk/util": ^0.8.2
     "@cardano-sdk/util-dev": ^0.8.0
+    "@dcspark/cardano-multiplatform-lib-nodejs": ^3.1.1
```

Comment:
There are no changes in package.json, please check if running `yarn` still adds this
```suggestion
```